ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt

```diff
@@ -65,6 +65,7 @@ enum class PageSet(
         )
     ),
     TIPS(
+        menu = R.menu.tutorial_tips_menu,
```

Comment:
I saw that the ticket mentioned also adding an exit option when going through the tutorial. Is the skip fine or should it be an X instead?

Reply:
I think having a skip action is fine

Reply:
wow, I evidently added the skip menu item, but forgot to include the menu in the tutorial page set. good catch that it already existed and just wasn't wired up